src/C2_economic_functions.py

```diff
@@ -82,10 +82,12 @@ def capex_from_investment(investment_t0, lifetime, project_life, discount_factor
         last_investment = first_time_investment / (
             (1 + discount_factor) ** ((number_of_investments - 1) * lifetime)
         )
+        # the residual of the capex at the end of the simulation time takes into
```

Comment:
Can you also add a hint of this procedure below the function definition in line 50? Ie. this:

`The specific capex is calculated by taking into account all future cash flows connected to the investment into one unit of the asset. This includes reinvestments, operation and management costs, dispatch costs as well as a deduction of the residual value at project end. The residual value is calculated with a linear depreciation of the last investment, ie. as a even share of the last investment over the lifetime of the asset. The remaining value of the asset is translated in a present value and then deducted.`

Reply:
done